spec/request/api/v1/targets/create_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+describe 'Create Targets', type: :request do
+  let!(:user) { create(:user, :confirmed) }
+  let!(:auth_header) { user.create_new_auth_token }
+  let(:target) { build(:target) }
+  let(:params) do
+    {
+      area_lenght: target.area_lenght,
+      title: target.title,
+      topic: target.topic,
+      latitude: target.latitude,
+      longitude: target.longitude
+    }
+  end
+
+  describe 'POST api/v1/targets' do
+    before do
+      post api_v1_targets_path, params: params, headers: auth_header, as: :json
+    end
+
+    context 'when the request is succesful' do
+      it 'is expected a successful response' do
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'is expected a increasement of targets count by 1' do
+        expect do
+          post api_v1_targets_path, params: params, headers: auth_header, as: :json
+        end.to change(Target, :count).by(1)
+      end
+
+      it 'is expected that response contains some body data' do
+        body = JSON response.body
+        expect(json_value(body, 'id')).not_to be_nil
+        expect(json_value(body, 'user_id')).not_to be_nil
+        expect(json_value(body, 'area_lenght')).to eq target.area_lenght
+        expect(json_value(body, 'title')).to eq target.title
+        expect(json_value(body, 'topic')).to eq target.topic
+        expect(json_value(body, 'latitude').to_d).to eq target.latitude
+        expect(json_value(body, 'longitude').to_d).to eq target.longitude
+      end
+    end
+
+    context 'when the user is not logged in' do
+      it 'is expected an unauthorized response' do
+        post api_v1_targets_path, params: params, as: :json
+        expect(response).to have_http_status(:unauthorized)
+      end
+    end
+
+    context 'when the request data is not valid' do
+      subject do
+        post api_v1_targets_path, params: params, headers: auth_header, as: :json
+      end
+
+      it 'requires the area_lenght parameter' do
+        params.delete :area_lenght
+        subject
+        expect(response).to have_http_status(:unprocessable_entity)
+      end
+
+      it 'requires the title parameter' do
```

Comment:
You are validating presence in the model spec, there's no need to test it here again.